src/main/resources/db/migration/V6.5.0.5.sql

```diff
@@ -0,0 +1,6 @@
+-- Store authenticated orcid in URL form
+ALTER TABLE authenticateduser ADD COLUMN IF NOT EXISTS authenticatedorcid VARCHAR(45);
+-- To do - add unique constraint in Postgres 15+
```

Comment:
I'm confused. We can't add the unique constraint on version of Postgres lower than 15?

Reply:
Earlier postgres can't have a unique constraint on a nullable field (or can't if you want to allow multiple rows with null and not just one).

Reply:
Awesome, thanks. Yeah, who knows what number! Lots of merging right now! 😅